dali/kernels/imgproc/color_manipulation/color_space_conversion_impl.h

```diff
@@ -130,6 +179,28 @@ DALI_HOST_DEV DALI_FORCEINLINE uint8_t rgb_to_cr(vec<3, uint8_t> rgb) {
   return ConvertSat<uint8_t>(dot(coeffs, rgb) + 128.0f);
 }
 
+template <typename Output, typename Input>
+DALI_HOST_DEV DALI_FORCEINLINE vec<3, Output> ycbcr_to_rgb(vec<3, Input> ycbcr_in) {
+  auto ycbcr = detail::norm(ycbcr_in);  // TODO(janton): optimize number of multiplications
+  float tmp_b = ycbcr.y - 0.5f;
+  float tmp_r = ycbcr.z - 0.5f;
+  auto r = ConvertSatNorm<Output>(ycbcr.x + 1.402f * tmp_r);
+  auto g = ConvertSatNorm<Output>(ycbcr.x - 0.34413629f * tmp_b - 0.71413629f * tmp_r);
+  auto b = ConvertSatNorm<Output>(ycbcr.x + 1.772f * tmp_b);
+  return {r, g, b};
+}
+
+template <>
+DALI_HOST_DEV DALI_FORCEINLINE vec<3, uint8_t> ycbcr_to_rgb(vec<3, uint8_t> ycbcr) {
+  float tmp_b = ycbcr.y - 128;
+  float tmp_r = ycbcr.z - 128;
+  auto r = clamp<uint8_t>(ycbcr.x + 1.402f * tmp_r, 0, 255);
+  auto g = clamp<uint8_t>(ycbcr.x - 0.34413629f * tmp_b - 0.71413629f * tmp_r, 0, 255);
+  auto b = clamp<uint8_t>(ycbcr.x + 1.772f * tmp_b, 0, 255);
```

Comment:
Likewise, this will truncate instead of rounding.

Reply:
Fixed